cobspec/src/main/java/com/saralein/cobspec/Main.java

```diff
@@ -62,4 +47,37 @@ public static void main(String[] args) {
 
         return new ArgsValidation(validators).validate(args);
     }
+
+    private static void createServer(Path root, int port, Logger logger, Path logTxt) {
```

Comment:
Would the name of this method change if the method returned an instance of `Application` that it did not create?